blinkit-app/src/homePage/molecules/mainPage/MainPage.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react'
+import Sidebar from '../sidebar/Sidebar'
```

Comment:
This is a molecule only for MainPage, create a molecules folder here and keep it with MainPage until any other component is using this